import-service/src/handlers/importFileParser.ts

```diff
@@ -0,0 +1,71 @@
+import { buildResponse } from '../utils';
+import { S3 } from 'aws-sdk';
+import csv from 'csv-parser';
+import { Readable } from 'stream';
+
+const s3 = new S3();
+const bucketName = process.env.BUCKET_NAME!;
+
+export const handler = async (event: any) => {
```

Comment:
**any** is very important bad habit to my mind!  
You have *noImplicitAny* in tsconfig, how ts has compiled this..?